ftt_ros:
  ros__parameters:
    params:
      use_tf: true
      map_frame: map
      robot_frame: base_link
      server_address: localhost:5000
      send_pose_period: 2.0
      send_map_period: 5.0
      image_buffer_size: 3
      image_buffer_step: 1.0
    topics:
      robot_mode: robot_mode
      gps_fix: gps_fix
      local_pose: local_pose
      map: map
      map_jpeg: map_jpeg
      image: image
      image_compressed: image_compressed